llpc/test/shaderdb/ray_tracing/standalone.rcall

```diff
@@ -0,0 +1,14 @@
+// BEGIN_SHADERTEST
+/*
+; RUN: amdllpc -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST: _call1:
+; SHADERTEST: AMDLLPC SUCCESS
+*/
+// END_SHADERTEST
+
+#version 460
```

Comment:
Nothing related to your change, but I see that the GitHub UI is able to syntax-highlight some file endings like the `.rchit` properly, while it doesn't for file endings like `.rcall`. Can you try forcing the syntax highlighting for these endings, so we don't end up having a inconsistent look when reviewing new lit tests in the future? A quick Google search yielded that this might be possible with .gitattributes (https://github.com/github-linguist/linguist/blob/master/docs/overrides.md).

Reply:
Should now work if you rebase this PR on dev.